specification/purview/data-plane/Azure.Analytics.Purview.DataMap/stable/2023-09-01/purviewdatamap.json

```diff
@@ -214,1406 +283,1499 @@
+            "name": "typeName",
+            "in": "path",
+            "description": "The name of the type.",
             "required": true,
             "type": "string"
           },
           {
-            "in": "body",
-            "name": "body",
-            "description": "The value of the attribute.",
-            "required": true,
-            "schema": {
-              "type": "object"
-            }
+            "name": "minExtInfo",
+            "in": "query",
+            "description": "Whether to return minimal information for referred entities.",
+            "required": false,
+            "type": "boolean"
+          },
+          {
+            "name": "ignoreRelationships",
+            "in": "query",
+            "description": "Whether to ignore relationship attributes.",
+            "required": false,
+            "type": "boolean"
+          },
+          {
+            "name": "attr_N:qualifiedName",
+            "in": "query",
+            "description": "Qualified name of an entity. E.g. to find 2 entities you can set\nattrs_1:qualifiedName=db1@cl1&attrs_2:qualifiedName=db2@cl1. (This is only an\nexample. qualifiedName can be changed to other unique attributes)",
+            "required": false,
+            "type": "string"
           }
         ],
         "responses": {
           "200": {
-            "description": "Updated with EntityMutationResponse.",
+            "description": "The request has succeeded.",
             "schema": {
-              "$ref": "#/definitions/EntityMutationResponse"
+              "$ref": "#/definitions/AtlasEntitiesWithExtInfo"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "An unexpected error response.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/AtlasErrorResponse"
             }
           }
-        }
-      },
-      "delete": {
-        "tags": [
-          "Entity"
-        ],
-        "description": "Delete an entity identified by its GUID.",
-        "operationId": "Entity_Delete",
+        },
         "x-ms-examples": {
-          "Entity_Delete": {
-            "$ref": "./examples/Entity_Delete.json"
+          "Entity_ListByUniqueAttributes": {
+            "$ref": "./examples/Entity_ListByUniqueAttributes.json"
           }
-        },
+        }
+      }
+    },
+    "/atlas/v2/entity/businessmetadata/import": {
+      "post": {
+        "operationId": "Entity_ImportBusinessMetadata",
+        "description": "Upload the file for creating Business Metadata in BULK",
+        "consumes": [
+          "multipart/form-data"
+        ],
         "parameters": [
           {
-            "$ref": "#/parameters/guid"
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "file",
```

Comment:
This name changed from the previous swagger?

Reply:
We notice in python SDK that the name used here will be the default key name. As in the API, the key name is file, so we think it is better to rename to file.
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/42c487ba-6f34-4eef-8ff0-ab62d1604cc1)
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/19d59d64-e88c-4be5-b0aa-3ff2c2640395)